buildpack/buildpack.go

```diff
@@ -58,6 +61,15 @@ func (b Buildpack) Dependencies() (Dependencies, error) {
 	return dependencies, nil
 }
 
+func (b Buildpack) DefaultVersion(id string) string {
+	defaults, ok := b.Metadata[DEFAULT_DEPENDENCIES].(map[string]interface{})
+	if !ok {
+		return ""
+	}
+
+	return defaults[id].(string)
```

Comment:
You cannot safely cast this value (e.g. could result in a panic if the value was an `int`).  Elsewhere we [return empty if the structure is completely wrong](https://github.com/cloudfoundry/libcfbuildpack/blob/94e191f14d37687c0ec6b204e17ac9ab1dc808e9/buildpack/buildpack.go#L80-L83), but [throw a descriptive error if only the value type is wrong](https://github.com/cloudfoundry/libcfbuildpack/blob/94e191f14d37687c0ec6b204e17ac9ab1dc808e9/buildpack/buildpack.go#L87-L90).  I'd recommend the same here unless you've got a compelling reason to do otherwise.